### PR TITLE
Updated yaml and added a couple of notes for clarity

### PR DESCRIPTION
**Pull request by gtrivedi88**
Purpose: To resolve [[SRVKP-4398](https://issues.redhat.com/browse/SRVKP-4398)]

Aligned team: DevTools

Version for cherrypicking: pipelines-docs-1.14

Content for preview: https://75334--ocpdocs-pr.netlify.app/openshift-pipelines/latest/secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements.html

SME review: @karthikjeeyar 

QE review: @ppitonak

Peer review:

### CODE REVIEW

File: modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc [hunk 1]
```diff
@@ -66,7 +71,7 @@ spec:
   results:
     - description: CVE result format  # <4>
       name: SCAN_OUTPUT
-      type: string
+      value: string
```
Comment (ppitonak) [inline modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc:74]: 1. `type` is the correct key
2. string is default value so we can omit this line completely

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc (hunk 1 above)
Comment (gtrivedi88) [reply modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc]: @ppitonak Updated. Could you please review and add LGTM or if you have anyother feedback please let me know?

**Review by ppitonak — APPROVED** [review]

File: modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc [hunk 2]
```diff
@@ -116,6 +122,11 @@ spec:
 <7> The tool-specific environment variables.
 <8> The shell script to be executed with json output. For example, scan_output.json.
 <9> The format to extract vulnerability summary (adjust `jq` command for different JSON structures).
++
+[NOTE]
+====
+This is just an example configuration. Modify the values according to your specific scanning tool to set results in the expected format.
```
Comment (tmalove) [inline modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc:128]:
```suggestion
This is an example configuration. Modify the values according to your specific scanning tool to set results in the expected format.
```

File: modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc [hunk 3]
```diff
@@ -15,6 +15,11 @@ You can also review the vulnerabilities in the *Vulnerabilities* column in the p
 .Viewing vulnerabilities on the `PipelineRun` list view
 image::vulnerabilities_list.png[]
 
+[NOTE]
+====
+Visual representation of identified vulnerabilities is available starting from the Openshift version 4.15 release.
```
Comment (tmalove) [inline modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc:20]: Use the attribute name for 'OpenShift Container Platform'.

**Review by tmalove — APPROVED** [review]

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM; merging.

**Pull request merged**